src/components/user/unavailability/unavailability.pg.repository.ts

```diff
@@ -74,6 +74,48 @@ export class UnavailabilityPgRepository extends DtoRepository(Unavailability) {
     return rows;
   }
 
+  async getUserIdByUnavailability(id: ID) {
+    const rows = await this.pg.query<{ id: ID }>(
+      `
+      SELECT admin_people_id as "id"
+      FROM sc.person_unavailabilities
+      WHERE id = $1;
+      `,
+      [id]
+    );
+    if (!rows) {
+      throw new NotFoundException(`Could not find unavailability id ${id}`);
+    }
+    return rows[0];
+  }
+
+  async list(
+    input: UnavailabilityListInput,
+    _session: Session
+  ): Promise<PaginatedListType<UnsecuredDto<Unavailability>>> {
+    const limit = input.count;
+    const offset = (input.page - 1) * input.count;
+    const [{ count }] = await this.pg.query<{ count: string }>(
+      'SELECT count(*) FROM sc.person_unavailabilities;'
+    );
+
+    const rows = await this.pg.query<UnsecuredDto<Unavailability>>(
+      `
+      SELECT id, admin_people_id, period_start as "start", period_end as "end", description, 
+      created_at as "createdAt", created_by_admin_people_id as "creator",
```

Comment:
Same here.

Reply:
just pushed commit [e3bc2fb] that removes the unnecessary fields